execwrap/execwrap_test.go

```diff
@@ -308,6 +308,8 @@ func TestRunCommandStdoutToFile(t *testing.T) {
 		})
 		defer os.Remove("/tmp/TestRunCommandStdoutToFile")
 
+		mock.EXPECT().Remove(gomock.Eq("filename")).Return(nil)
```

Comment:
Does this make sense as a separate test? I usually write more BDD "one expect per test"-style tests, but that might not be the best path forward on go. Just a thought.